src/OSPSuite.UI/Controls/UxHtmlLabel.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.Drawing;
+using DevExpress.Utils.Controls;
+
+namespace OSPSuite.UI.Controls
+{
+   public class UxHtmlLabel : UxRichEditControl, IXtraResizableControl
+   {
+      private static readonly object _layoutInfoChanged = new object();
+      private static string _fontFamily;
+      private static double _fontSize;
+
+      public UxHtmlLabel(string fontFamily, double fontSize)
+      {
+         _fontSize = fontSize;
+         _fontFamily = fontFamily;
+      }
+
+      public UxHtmlLabel() : this("Tahoma", 8.25)
+      {
+
+      }
+
+      event EventHandler IXtraResizableControl.Changed
+      {
+         add
+         {
+            Events.AddHandler(_layoutInfoChanged, value);
+         }
+         remove
+         {
+            Events.RemoveHandler(_layoutInfoChanged, value);
+         }
+      }
+      protected void RaiseChanged()
+      {
+         var changed = (EventHandler)Events[_layoutInfoChanged];
+         changed?.Invoke(this, EventArgs.Empty);
+      }
+
+      bool IXtraResizableControl.IsCaptionVisible => false;
+
+      Size IXtraResizableControl.MaxSize => Size;
+
+      Size IXtraResizableControl.MinSize => Size;
+
+      protected override void OnSizeChanged(EventArgs e)
+      {
+         base.OnSizeChanged(e);
+         RaiseChanged();
+      }
+
+      public void Caption(string caption, string fontFamily = "", double? fontSize = 8.25)
+      {
+          Document.HtmlText = stylizeHtmlText(fontFamily, fontSize) + $"<div>{caption}</div>";
+      }
+
+      private static string stylizeHtmlText(string fontFamily, double? fontSize)
+      {
+         return $@"
+            <style>
+               div {{
```

Comment:
Yeah, you can use string interpolation. Just need to double up the curly braces.